plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerArtifact.kt

```diff
@@ -99,6 +101,14 @@ open class CodeModernizerArtifact(
             if (!patchesDir.isDirectory()) {
                 throw RuntimeException("Expected root for patches was not a directory.")
             }
+            val patch = patchesDir.walk().map { fileSystem.findFileByNioFile(it) }.toList()
+            if (patch[0] == null) {
+                ApplicationManager.getApplication().invokeLater {
+                    ApplicationManager.getApplication().runWriteAction {
+                        VirtualFileManager.getInstance().syncRefresh() // try refreshing Virtual File System if diff.patch not found
+                    }
+                }
+            }
             return patchesDir.walk()
                 .map { fileSystem.findFileByNioFile(it) ?: throw RuntimeException("Could not find patch") }
```

Comment:
This exception was being thrown repeatedly, as seen by internal and external users and our dashboard.